src/components/NavBar/NavBar.stories.js

```diff
@@ -1,11 +1,18 @@
 import React from 'react';
+import { storiesOf } from '@storybook/react';
+import StoryRouter from 'storybook-react-router';
 import { NavBar } from './NavBar';
 
 export default {
   component: NavBar,
   title: 'NavBar',
 };
 
-export const guestNavBar = () => <NavBar isAuthenticated={false} />;
-
-export const userNavBar = () => <NavBar isAuthenticated />;
+storiesOf('NavBar', module)
+  .addDecorator(StoryRouter())
+  .add('guest', () => (
+    <NavBar isAuthenticated={false} />
```

Comment:
If authenticated is false by default, then we don't need to explicitly set it to false